migrations/server/mysql/0004_drop_timestamp_key.up.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE `timestamp_keys`;
```

Comment:
Non-blocking - this is probably a bad idea from a historical point of view, but wondering if it's better to also remove the table from 0001 and add a `IF EXISTS` to this drop table command, that way it doesn't have to be created and then immediately dropped.

https://dev.mysql.com/doc/refman/5.7/en/drop-table.html to this if there is no error, and modify